requirements/flocker.txt.in

```diff
@@ -27,17 +27,21 @@ netifaces
 python-novaclient
 # FLOC-4411 oauth2client 2 no longer exports SignedJwtAssertionCredentials
 oauth2client < 2.0.0
+# XXX This is here to ensure that buildbot installs a version of pip and
+# setuptools capable of parsing pkg_resources environment markers
+# e.g. nomenclature; sys_platform==linux2 (FLOC-4429)
+pip >= 8.1.2
```

Comment:
This is a hack to ensure that buildbot can install the `[dev]` requirements when it build packages.
Without this, buildbot is installing the dev requirements using pip-7.1.0 which I think must interact differently with the new version of setuptools introduced in this branch.